doc/catppuccin.txt

```diff
@@ -118,7 +118,7 @@ options and settings.
             -- For more plugins integrations please scroll down (https://github.com/catppuccin/nvim#integrations)
         },
     })
-    
```

Comment:
They're automatically generated by the vimdoc bot right? I guess these whitespaces will be back some time in the future.